roles/ovirt-manageiq/README.md

```diff
@@ -0,0 +1,105 @@
+| Name               | Default value     |  Description                                    |
+|--------------------|-------------------|-------------------------------------------------|
+| ovirt_user         | UNDEF             | Define the user to be used to access oVirt/RHV. |
+| ovirt_password     | UNDEF             | Define password of the 'ovirt_user'.            |
+| ovirt_fqdn         | UNDEF             | Define the FQDN of the oVirt/RHV.               |
+| ovirt_ca           | UNDEF             | Define the path to the CA of the oVirt/RHV.     |
+
+Virtual Machine variables:
+| Name               | Default value     |  Description                                 |
+|--------------------|-------------------|----------------------------------------------|
+|  miq_vm_name       | manageiq_euwe     | Define the name of the ManageIQ/CloudForms VM in oVirt/RHV.|
+| miq_vm_cluster     | Default           | Define the cluster of the virtual machine    |
+| miq_vm_memory      | 6GiB              | Define the system memory of the virtual machine    |
+| miq_vm_cpu         | 2                 | Define the number of virtual machine CPU cores   |
+| miq_vm_os          | rhel_7x64         | Define the operating system of the virtual machine |
+| miq_vm_cloud_init  | UNDEF             | Define cloud init dictionary to be passed to virtual machine. |
+
+Virtual Machine disks variables:
+| Name                | Default value     |  Description                                         |
+|---------------------|-------------------|------------------------------------------------------|
+| miq_vm_disk_name    | `miq_vm_name`     | Define name of the virtual machine disk.             | 
+| miq_vm_disk_storage | UNDEF             | Define the target storage domain of the disk.        |
+| miq_vm_disk_size    | 50GiB             | Define the size of the virtual machine disk.         |
+| miq_vm_disk_interface | virtio          | Define the interface of the virtual machine disk.    |
+| miq_vm_disk_format | cow                | Define the format of the virtual machine disk.       |
+
+
+Virtual Machine nics variables:
+| Name                | Default value     |  Description                                         |
+|---------------------|-------------------|------------------------------------------------------|
+| miq_vm_nics         | {'name': 'nic1', 'profile_name': 'ovirtmgmt', 'interaface': 'virtio'} | Dictionary variable which define network interfaces for virtual machine. |
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+Note that for passwords you should be using Ansible vault.
+
+```yaml
+    - name: Deploy ManageIQ to oVirt engine
+      hosts: localhost
+      gather_facts: no
+
+      vars:
+        ovirt_fqdn: ondra.local
+        ovirt_password: 123456
+        ovirt_username: admin@internal
+
+        miq_vm_name: manageiq_euwe
+        miq_qcow_url: http://releases.manageiq.org/manageiq-ovirt-euwe-3.qc2
```

Comment:
Does ManageIQ provide some link to latest release, so we won't need to update this often?